Sources/AppcuesKit/Presentation/Debugger/PushVerifier.swift

```diff
@@ -45,14 +57,22 @@ internal class PushVerifier {
                 return "Error 7: Receive completion called too many times"
             case .noSDKResponse:
                 return "Error 8: Receive response not passed to SDK"
+            case .noForegroundPresentationHandler:
+                return "Error 9: Foreground presentation handler not implemented"
+            case .noForegroundPresentationOption:
+                return "Note: Notifications won't display while app is in foreground"
```

Comment:
just wondering if there is some way to make it clear that this is not our SDK imposing a limitation or something - want the app dev to know that this is due to how they set up their app, not a limitation in Appcues.

maybe:
```
Note: Application has not been configured to display foreground notifications.
```